CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 ## 7.5.0 (Unreleased)
 
+BREAKING CHANGES:
+* netapp: changed `peerIpAddresses` field type from String to Array in `google_netapp_volume` resource ([#10757](https://github.com/hashicorp/terraform-provider-google-beta/pull/10757))
```

Comment:
```suggestion
* netapp: changed `peerIpAddresses` field type from String to Array in `google_netapp_volume` resource, as it was unusable otherwise ([#10757](https://github.com/hashicorp/terraform-provider-google-beta/pull/10757))
```